### ISSUE TITLE
Player texture upside down when moving left.

### ISSUE THREAD

**Issue by colbySullivan** [kaissouDev/Eat-the-Mushroom#19]
Not sure if this is a design choice but I wanted to submit an issue to see your opinion. If you want I can add a fix if you want it.

**Comment by kaissouDev** [comment]
yes, it's a small problem that didn't bother me, but if you can fix it I won't say no.

**Comment by colbySullivan** [comment]
I think I have a fix for it but there is a bug when the player presses the arrow keys too fast. Since its a smaller issue I don't think its worth sending a PR.

**Comment by kaissouDev** [comment]
Its possible then the bug Comes from raylib

**Comment by kaissouDev** [comment]
I think we can wait a while before correcting the problem.
